controllers/glance_controller.go

```diff
@@ -655,6 +658,10 @@ func (r *GlanceReconciler) reconcileNormal(ctx context.Context, instance *glance
 	if err != nil {
 		return ctrl.Result{}, err
 	}
+
+	// remove finalizers from unused MariaDBAccount records
+	mariadbv1.DeleteUnusedMariaDBAccountFinalizers(ctx, helper, glance.DatabaseName, instance.Spec.DatabaseAccount, instance.Namespace)
```

Comment:
this is working, and mariadb-operator will not emit DROP USER until the Glance finalizer (and all other finalizers that are not MariaDBAccount itself) are removed